app/assets/javascripts/components/tickets/tickets_handler.jsx

```diff
@@ -53,9 +65,12 @@ export class TicketsHandler extends React.Component {
       }
     };
 
+    const TICKETS_PER_PAGE = 10;
+
     const elements = this.props.filteredTickets.map(ticket => (
       <Row key={ticket.id} ticket={ticket} />
-    ));
+    )).slice(this.state.page, this.state.page + TICKETS_PER_PAGE);
+    const pagesLength = Math.floor(elements.length / TICKETS_PER_PAGE) + 1;
```

Comment:
This should be done based on the full filteredTickets array, not just the slice of elements on the current page. I think it also shows an extra page when the ticket count is a multiple of ten. Maybe should be `(count - 1) / TICKETS_PER_PAGE` ?